lbrynet/schema/validator.py

```diff
@@ -17,7 +17,7 @@
 
 
 def validate_claim_id(claim_id):
-    if not len(claim_id) == 40:
+    if not len(claim_id) <= 40:
```

Comment:
Needs to check for full 40 char length.

Reply:
I'm adding a `for_resolve` flag in this function.